pkgs/games/gzdoom/default.nix

```diff
@@ -83,6 +83,10 @@ let
       runHook postInstall
     '';
 
+    fixupPhase = ''
+      wrapProgram $out/lib/gzdoom/gzdoom --prefix PATH ":" ${lib.makeBinPath [ plasma5Packages.kdialog ]}
```

Comment:
maybe we want /run/current-system/sw/bin/ here? this would probably be a bit impure but we wouldn't increase the closure size by default

Reply:
Correct me if I'm wrong, but gzdoom would still crash with `kdialog not found` if kdialog is not installed. I.e. for anyone not on KDE. I don't think that is something we want.

Reply:
it should not crash, at least it does not crash for me. I believe the kdialog not found is just a warning. having gzdoom depend on kdialog would pull like 500MB of additional dependencies down if you haven't had KDE installed already, thats a lot of overhead

Reply:
It does when it can't find any IWAD files. See [the issue that prompted this change](https://github.com/NixOS/nixpkgs/issues/154918). (Depends on your definition of "crashing". The program exits without any further information, I don't think that's expected behaviour)

Reply:
Maybe that is something I should take upstream. Quick search doesn't yield any related issues on their repo